tauri/src/styles/index.css

```
@@ -1,32 +1,10 @@
+@plugin "@iconify/tailwind4";
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    color: #f6f6f6;
-    background-color: #2f2f2f;
-  }
 
-  a:hover {
-    color: #24c8db;
-  }
 
```

ESLint

[error] Delete `⏎⏎⏎`

(format/prettier)